src/touchstone/compare.py

```diff
@@ -17,6 +17,37 @@
 _logger = logging.getLogger("touchstone")
 
 
+def emit_csv(values,args):
```

Comment:
This is specific to uperf. So this will break for other benchmarks.

Reply:
True...